hydra-cluster/src/Hydra/Cluster/Util.hs

```diff
@@ -75,24 +75,31 @@ chainConfigFor me targetDir nodeSocket hydraScriptsTxId them contestationPeriod
   when (me `elem` them) $
     failure $
       show me <> " must not be in " <> show them
-  readConfigFile ("credentials" </> skName me) >>= writeFileBS (skTarget me)
-  readConfigFile ("credentials" </> vkName me) >>= writeFileBS (vkTarget me)
+
+  moveFile me "vk"
+  moveFile me "sk"
+  moveFile (toExternalCommitActor me) "vk"
+  moveFile (toExternalCommitActor me) "sk"
+
   forM_ them $ \actor ->
-    readConfigFile ("credentials" </> vkName actor) >>= writeFileBS (vkTarget actor)
+    moveFile actor "vk"
   pure $
     Direct
       defaultDirectChainConfig
         { nodeSocket
         , hydraScriptsTxId
-        , cardanoSigningKey = skTarget me
-        , cardanoVerificationKeys = [vkTarget himOrHer | himOrHer <- them]
+        , cardanoSigningKey = actorFilePath me "sk"
+        , cardanoVerificationKeys = [actorFilePath himOrHer "vk" | himOrHer <- them]
         , contestationPeriod
         }
  where
-  skTarget x = targetDir </> skName x
-  vkTarget x = targetDir </> vkName x
-  skName x = actorName x <.> ".sk"
-  vkName x = actorName x <.> ".vk"
+  actorFilePath actor fileType = targetDir </> actorFileName actor fileType
+  actorFileName actor fileType = actorName actor <.> fileType
+
+  moveFile actor fileType = do
+    let fileName = actorFileName actor fileType
+        filePath = actorFilePath actor fileType
+    readConfigFile ("credentials" </> fileName) >>= writeFileBS filePath
```

Comment:
Should: rename this function to `copyFile`.

It does not move it, it writes a new one.